cfgov/youth_employment_success/jinja2/youth-employment-success/route-option-form.html

```diff
@@ -0,0 +1,92 @@
+{% from 'atoms/checkbox.html' import render with context %}
+{% set labels = ['Walk', 'Drive', 'Bike', 'Public transit', 'Get dropped off'] %}
+
+<form class="o-yes-route-option">
+  <span class="a-label__heading a-flush__bottom"><b>What type of transportation will you take?</b></span>
+  <span class="a-label__flush a-label_helper__block a-micro-copy">Choose one</span>
+  {% for label in labels|batch(2) %}
```

Comment:
TRIVIAL: We try to add spaces between the pipe to make it stand out better 

```
{% for label in labels | batch( 2 ) %}
```

Here and below.

Reply:
Ah good to know. All the examples I saw on the internet seem to have the pipe jammed up against the arguments.

Reply:
Yeah for better or worse we tend toward spaces inside things, though it's not totally consistent.